configurations/force-gossip-topology-changes.yaml

```diff
@@ -1,3 +1,4 @@
 append_scylla_yaml:
   force_gossip_topology_changes: true
   enable_tablets: false
+  tablets_mode_for_new_keyspaces: 'disabled'
```

Comment:
Did you test this config?
Scylla could failed to start if unsupported parameter in scylla.yaml or passed as parameter